.github/workflows/deploy-auto.yml

```diff
@@ -0,0 +1,21 @@
+# Automatically create a new version/tag, if new commits are available since the last tag
+name: Auto-tagger
+
+on:
+  schedule:
+    - cron: '51 15 */4 * *' # Every 4 days at 15:15 https://crontab.guru/#51_15_*/4_*_*
+
+jobs:
+  create-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: fregante/setup-git-token@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - run: |
+          if [[ $(git tag -l --points-at HEAD) = "" ]]; then
+            export VER=$(npx daily-version)
```

Comment:
Versioning here is different from https://github.com/sindresorhus/refined-github/pull/2945/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R15. Should use either `20.x.x` or `2020.x.x`.

Reply:
Good catch, it was a mistake. We have `20.x.x` versions but npmhub has `2020.x.x` versions. I had copied it from there.